.github/ISSUE_TEMPLATE/v2-to-v3-migration.yml

```
@@ -0,0 +1,77 @@
+---
+name: "🔀 Migration Issue: AWS SDK JS v2 to v3"
+description: Report an issue or discrepancy encountered during migration from AWS SDK JS v2 to v3
+title: "MIGRATION ISSUE: [Your Title Here]"
+labels: [needs-triage, v2-v3-inconsistency]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Migration Issue from AWS SDK JS v2 to v3
+        Thank you for taking the time to report your migration issue. To help us address your concerns effectively, please provide as much detail as possible.
+
+  - type: checkboxes
+    attributes:
+      label: Pre-Migration Checklist
+      options:
+        - label: I've read the [Migration Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/migrating-to-v3.html).
+          required: true
+        - label: I've reviewed the upgrading notes and major version differences mentioned in [`UPGRADING.md`](https://github.com/aws/aws-sdk-js-v3/blob/main/UPGRADING.md).
+          required: true
+        - label: I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js) for similar migration issues.
+          required: true
+
+  - type: dropdown
+    id: runtime
+    attributes:
+      label: Which JavaScript Runtime is this issue in?
+      options:
+        - Node.js
+        - Browser
+        - Lambda
```

Users might get confused between Node.js and Lambda.
```suggestion
        - Node.js
        - Browser
```

---

If you really want to include the word Lambda, it could be 

`Node.js (includes AWS Lambda)`

or add a yes or no question

> if you selected Node.js, is your application running on AWS Lambda?

---

```suggestion
        - Node.js (includes AWS Lambda)
        - Browser
```

---

I often find myself 5-6 back and fourth messages deep into a thread conversation to only then realize customers are talking about Lambda specific issues. 

I think having Lambda as an option is super helpful for OSDS to form an opinion right from the get-go.

How about this?

```
        - Node.js
        - Node.js on Lambda
        - Browser
        - React Native
```